sale_purchase_force_vendor/models/stock_move.py

```diff
@@ -1,22 +1,48 @@
 # Copyright 2022 Tecnativa - Víctor Martínez
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
-from odoo import models
+from odoo import api, fields, models
 
 
 class StockMove(models.Model):
     _inherit = "stock.move"
 
+    propagated_sale_line_id = fields.Many2one(
```

Comment:
@marielejeune For that part, you need to use (and migrate) this : https://github.com/OCA/sale-workflow/tree/14.0/sale_order_line_chained_move

Reply:
@rousseldenis Done